allo/backend/hls.py

```diff
@@ -1,6 +1,7 @@
 # Copyright Allo authors. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
-# pylint: disable=consider-using-with, no-name-in-module, too-many-branches
+# pylint: disable=consider-using-with, no-name-in-module, too-many-branches, \
+# pylint: disable=too-many-instance-attributes, too-many-arguments
```

Comment:
Avoid using global exceptions. Attach it to the corresponding line that prompts the error.